telegram/ext/_application.py

```diff
@@ -1076,6 +1082,7 @@ async def _update_fetcher(self) -> None:
                     self.create_task(
                         self.__process_update_wrapper(update),
                         update=update,
+                        name=f"Application:process_concurrent_update_{self.update_queue.qsize()}",
```

Comment:
bot.id for consistency? Why the updates queue size? I would find the update ID more helpful...

Reply:
> I would find the update ID more helpful

I wanted to do that, but the update queue can contains updates of other types also (like in the customwebhookbot example), so that would result in an error.

The only other thing which makes sense is using the queue size

Reply:
Hah, very good point! Tbh I don't see what the queue size would be good for though. It doesn't help identifying which update was processed... One could use the ID if available and the type otherwise. Or we leave out both

Reply:
i'll leave both then